roles/skills/files/git-commit/SKILL.md

```
@@ -0,0 +1,253 @@
+9. Agent commits: must NOT include `Co-Authored-By:` footer
+
+### Body (optional)
+
+- Use imperative, present tense
+- Explain the motivation for the change
+- Contrast with the previous behaviour
+
+### Valid examples
+
+Simple feature:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)